tests/test_lab_dev/test_states/test_quadrature_eigenstate.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the ``SqueezedVacuum`` class."""
```

Comment:
```suggestion
"""Tests for the ``QuadratureEigenstate`` class."""
```

Actually I almost want to start removing these strings altogether since test files should be written in a way thats pretty self explanatory